trackintel/model/trips.py

```diff
@@ -61,6 +65,15 @@ def _validate(obj):
             obj["finished_at"]
         ), "dtype of finished_at is {} but has to be datetime64 and timezone aware".format(obj["finished_at"].dtype)
 
+        # Check geometry if Trips is a GeoDataFrame
+        if "geom" in obj.columns:
+            # check geometry
+            assert obj.geometry.is_valid.all(), (
+                "Not all geometries are valid. Try x[~ x.geometry.is_valid] " "where x is you GeoDataFrame"
+            )
+            if obj.geometry.iloc[0].geom_type != "MultiPoint":
+                raise AttributeError("The geometry must be a MultiPoint (only first checked).")
```

Comment:
Can you add a test for that special case? Can be a small one with a dummy example

Reply:
I added this test in `tests/model/test_trips.py`, see below